chunky/src/java/se/llbit/chunky/ui/render/tabs/LightingTab.java

```diff
@@ -133,10 +136,36 @@ public LightingTab() throws IOException {
     drawSun.setTooltip(new Tooltip("Draws the sun texture on top of the skymap."));
 
     sunSamplingStrategy.getItems().addAll(SunSamplingStrategy.values());
+    // Hide HIGH_QUALITY in the GUI but leave it available through JSON editing/loading existing scenes
+    sunSamplingStrategy.getItems().remove(SunSamplingStrategy.HIGH_QUALITY);
```

Comment:
As of #1681, `SunSamplingStrategy` can now implement `getDeprecationStatus` to return `HIDDEN` if it is `HIGH_QUALITY`. The UI logic will then be implemented at a later point (for all selectors at once).

```suggestion
```